### PR TITLE
Adding corrections to portuguese and missing translations

### PR DESCRIPTION
**Pull request by thomasmodeneis**
1) Adding corrections to portuguese and missing translations
2) Adding puppet-vm section to the virtual_machines in english and pt
3) Adding thanks to SouJava for the translations on thanks_and_support page and doing translations to portuguese

### CODE REVIEW

**Comment by thomasmodeneis** [comment]
Mani can you please review if is ok to add the thanks section and I will merge the pull request.
Thanks.

**Comment by thomasmodeneis** [comment]
Resolves #5

File: en/thanks_and_support.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ Thanks to **London Java Community**, its founder [Barry Cranford](http://twitter
 
 Also you would like to know that [RecWorks](http://www.recworks.co.uk/) has been supporting LJC with all events and presentation activities, since the inception of the community.
 
+Thanks to [SouJava](http://soujava.org.br/) for helping us to get this book translated into Portuguese.
```
Comment (neomatrix369) [inline en/thanks_and_support.md]: Thats good, im glad we can mention names of jugs in our books.

File: en/virtual-machines/adoptjdk_puppet_vm.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# OpenJDK 8 Vagrant Puppet
```
Comment (neomatrix369) [inline en/virtual-machines/adoptjdk_puppet_vm.md]: I trust these steps work, so i'm happy to approve them.

File: en/virtual-machines/adoptjdk_puppet_vm.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+```
+    $ vagrant up
+```
+
+Puppet will install Mercurial and OpenJDK 7.
+
+Clone OpenJDK 8 project.
+
+Get get_source.sh in order to download all OpenJDK 8 sources.
```
Comment (neomatrix369) [inline en/virtual-machines/adoptjdk_puppet_vm.md]: try using this kind of formatting for file names, commands and important keywords, for e.g.
```get_source.sh```

Comment (neomatrix369) [inline en/virtual-machines/adoptjdk_puppet_vm.md]: You have done that for ```vagrant up``` so please apply the same style.

Comment (thomasmodeneis) [inline en/virtual-machines/adoptjdk_puppet_vm.md]: Sure I have just fixed this.
Submiting another commit.

**Pull request merged**